elasticsearch/docker/templates/Dockerfile.j2

```diff
@@ -47,12 +47,17 @@ RUN yum -y update \
     && yum install -y unzip glibc.x86_64 cmake \
     && yum clean all
 
-RUN groupadd -g 1000 elasticsearch && \
-    adduser -u 1000 -g 1000 -d /usr/share/elasticsearch elasticsearch
+ARG DEFAULT_UID=1000
+ARG DEFAULT_GID=1000
+ENV DEFAULT_UID $DEFAULT_UID
+ENV DEFAULT_GID $DEFAULT_GID
 
-USER 1000
+RUN groupadd -g $DEFAULT_GID elasticsearch && \
+    adduser -u $DEFAULT_UID -g $DEFAULT_GID -d /usr/share/elasticsearch elasticsearch
```

Comment:
the elasticsearch user and group will be created with DEFAULT_UID and DEFAULT_GID, respectively